.aegir.js

```diff
@@ -0,0 +1,31 @@
+const replaceNodeBuiltIns = () => {
+  const replace = {
+    'os': require.resolve('os-browserify')
```

Comment:
replacing `os` dependency with https://github.com/CoderPuppy/os-browserify/blob/master/browser.js which basically brings in sensible defaults. ESbuild recommends writing a custom plugin for this replacement.

Reply:
Thanks @lidel I'll add a build action for ipfs-geoip to compile the dependency.

Reply:
@whizzzkid just keep in mind that our packages are going esm-only and we should stick to that path for changes moving forward. esm-only doesn't necessarily mean no-build, but if we're building, we need to make sure to export esm

Reply:
@SgtPooki @lidel done, `ipfs-geoip@9.*` is now a pure ESM module.